code/game/objects/structures/tables_racks.dm

```diff
@@ -208,6 +214,35 @@
 	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
 	buildstack = null //No buildstack, so generate from mat datums
 
+///Table on wheels
+/obj/structure/table/rolling
+	name = "Rolling table"

```

Comment:
Add a description here, since the parent description makes no sense. Something will like "A NT brand "Rolly poly" rolling table. It can and will move."